src/components/Playlist/Playlist.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import {
+  useManifestState,
+  useManifestDispatch,
+} from '../../context/manifest-context';
+import {
+  usePlayerDispatch,
+  usePlayerState,
+} from '../../context/player-context';
+
+const getItemLabel = ({ label }) => {
+  return label[0];
+};
+
+const Playlist = ({ manifest = [] }) => {
+  const [activeItem, setActiveItem] = useState();
+  const dispatch = useManifestDispatch();
+  const manifestState = useManifestState();
+  console.log(manifestState);
+
+  // const playerState = usePlayerState();
+  // const playerDispatch = usePlayerDispatch();
+
+  const getPlaylistItems = ({ manifests }) => {
+    const manifestData = manifests.reduce(
+      (result, item) => [
+        ...result,
+        { id: item['@id'], label: getItemLabel(item) },
+      ],
+      []
+    );
+
+    return manifestData;
+  };
+
+  const fetchPlaylistManifest = ({ id: manifestUrl }) => {
+    fetch(manifestUrl)
+      .then((result) => result.json())
+      .then((data) => {
+        dispatch({ manifest: data, type: 'updateManifest' });
+        setActiveItem(manifestUrl);
+        console.log(data);
+
+        return data;
+      });
+  };
```

Comment:
I'd say for now we just hardcode some mock display data.  Not sure that we'll want to make an extra network request for each manifest in a Playlist manifest.  There's probably a better way to serve this data up for the front-end.